src/index.ts

```diff
@@ -54,9 +54,7 @@ const jongseongMap: Record<string, string> = {
 const endWith = {
   parenthesis: new RegExp(
     `${parenthesisToSkip
-      .map((parens) => {
-        const opening = parens[0],
-          closing = parens[1]
+      .map(([opening, closing]) => {
```

Comment:
I believe this is in its simplest form, but I doubt the clarity of this. Also does not support parenthesis are consisted of multiple characters; e.g. `{{ blah }}` or `<% blah %>`.

Reply:
Then how about to change structure of `parenthesisToSkip` field in `data.json` file?

Current:
```
parenthesisToSkip = ["[]", "()"]
parenthesisToSkip.map(([opening, closing]) => { /* ... */ })
```
To be:
```
parenthesisToSkip = [
    ["[", "]"],
    ["(", ")"]
]
parenthesisToSkip.map(([opening, closing]) => { /* ... */ })
```

We can adopt this structure without changing codes, and get more readability.